scripts/aws/setupdb.sh

```diff
@@ -19,5 +19,10 @@ envdir /etc/nyc-trees.d/env /opt/app/manage.py migrate
 # Load block face data
 envdir /etc/nyc-trees.d/env /opt/app/manage.py loaddata $DIR/../src/nyc_trees/apps/survey/fixtures/blockface.json
 
+# Setup soft launch flag if it does not already exist
+if [ envdir /etc/nyc-trees.d/env /opt/app/manage.py flag -l | grep full_access ]; then
```

Comment:
This did not create a flag for me. I think it is because the conditional is backwards. ``grep`` will exit non-zero if it does not find the text.